packages/math-input/src/components/key-handlers/handle-jump-out.ts

```diff
@@ -26,7 +26,7 @@ const KeysForJumpContext = {
  */
 function handleJumpOut(mathField: MathFieldInterface, key: Key): void {
     const cursor = getCursor(mathField);
```

Comment:
this is not specific to this change, but I noticed that we use cursor.parent.parent a lot in this switch statement. Maybe we can define it as a variable with a more descriptive name up here to make the code a little more readable.

Reply:
You're probably right, but I agree it's outside this set of changes and I'm too afraid of the code to try a quick fix.